src/main/java/seedu/address/model/module/Year.java

```diff
@@ -41,7 +51,17 @@ public Year(int year) {
      * @return true if given string is a valid year
      */
     public static boolean isValidYear(int year) {
-        return Integer.toString(year).matches(YEAR_VALIDATION_REGEX);
+        return isValidYear(Integer.toString(year));
```

Comment:
Actually will this potentially throw a exception when the year is actually invalid? (like if they enter a string instead of a number)

Reply:
`isValidYear` will return false if year is invalid